merkle-tree/indexed/src/lib.rs

```diff
@@ -27,8 +27,8 @@ pub mod zero_copy;
 
 use crate::errors::IndexedMerkleTreeError;
 
-pub const FIELD_SIZE_SUB_ONE: &str =
-    "21888242871839275222246405745257275088548364400416034343698204186575808495616";
+pub const HIGHEST_ADDRESS_PLUS_ONE: &str =
+    "452312848583266388373324160190187140051835877600158453279131187530910662655";
```

Comment:
What does this value stand for now? It seems lower than the Fr field size limit. Is it intentional to reduce the available range of addresses?

Reply:
- isLess limits the bitsize to 248 bit equivalent to truncated addresses of 31 bytes
  - our highest indexed array value FIELD_SIZE - 1 is too great now to work with the circuit, `HIGHEST_ADDRESS_PLUS_ONE` is `2^248 -1`the highest 248 bit value. The highest address must be less than `HIGHEST_ADDRESS_PLUS_ONE` hence the name plus one.